presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -480,7 +479,7 @@ private TestingPrestoSparkTaskExecutorFactoryProvider(String instanceId)
         }
 
         @Override
-        public IPrestoSparkTaskExecutorFactory get(SparkProcessType processType)
+        public IPrestoSparkTaskExecutorFactory get()
```

Comment:
why no longer need `processType` ?

Reply:
PrestoSparkTaskExecutorFactoryProvider is only used to create tasks on executor, so it is always `EXECUTOR`.